src/k8s/pkg/k8sd/api/cluster_remove.go

```diff
@@ -42,8 +42,8 @@ func (e *Endpoints) postClusterRemove(s *state.State, r *http.Request) response.
 		log.Info("Waiting for node to not be pending")
 		control.WaitUntilReady(ctx, func() (bool, error) {
 			var notPending bool
-			if err := s.Database.Transaction(ctx, func(ctx context.Context, tx *sql.Tx) error {
-				member, err := cluster.GetInternalClusterMember(ctx, tx, req.Name)
+			if err := s.Database().Transaction(ctx, func(ctx context.Context, tx *sql.Tx) error {
+				member, err := cluster.GetCoreClusterMember(ctx, tx, req.Name)
```

Comment:
what's this change do?

`GetInternalClusterMember` -> `GetCoreClusterMember`

Reply:
The microcluster team renamed their core implementation from `internal` to `core`. This function is provided by microcluster and just got renamed as well.
No functional changes.